bigquery/samples/tests/test_table_samples.py

```diff
@@ -15,11 +15,10 @@
 from .. import create_table
 
 
-def test_table_samples(capsys, client, project_id, dataset_id, table_id):
-    """Since creating a table is a long operation, test all table model samples
+def test_table_samples(capsys, client, random_table_id):
```

Comment:
This function can be renamed to `test_create_table`.

Reply:
Changed the name as directed.